assets/js/blocks/product-template/index.tsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import { BlockConfiguration, registerBlockType } from '@wordpress/blocks';
+import { isExperimentalBuild } from '@woocommerce/block-settings';
+
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+import edit from './edit';
+import save from './save';
+import icon from './icon';
+
+const blockConfig: BlockConfiguration = {
+	...metadata,
+	icon,
+	edit,
+	save,
+};
+
+if ( isExperimentalBuild() ) {
+	registerBlockType( metadata.name, blockConfig );
+}
```

Comment:
You can use `registerExperimentalBlockType` function available in:

`import { registerExperimentalBlockType } from '@woocommerce/block-settings';`

Although it's basically doing the same thing, so leaving the decision up to you.

Reply:
I'm going to go ahead and use registerExperimentalBlockType to make the code a bit cleaner. If there's an abstraction available, I think it's better to use it. Thanks Karol.